### ISSUE TITLE
Feedbin Sync

### ISSUE THREAD

**Issue by nonjo** [yang991178/fluent-reader#251]
I have tried both direct download and Microsoft Store versions of this app, and while it imports the names of the sources and the groups I've made in Feedbin, it either downloads no articles or a small handful of articles.

**Comment by GiovanH** [comment]
If it imported the names and sources correctly, this probably isn't a sync issue. If you add those same sources manually, how many entries load?

**Comment by nonjo** [comment]
I exported an OPML of the feeds from NetNewsWire on my Mac and imported the
feeds into Fluent Reader. While I did get an error message regarding a
number of feeds, most were imported and the articles seem to be loading
correctly.
--

Alex

**Comment by hyliu1107** [comment]
Feeds imported through OPML are stored locally in the app. You need to go to "settings > services" and log into Feedbin to enable sync.

**Comment by yang991178** [comment]
Feeds imported through OPML are stored locally. You need to log into Feedbin through "settings > services" to enable sync.

**Comment by nonjo** [comment]
Yes, I understand. I had Feedbin sync set up and reported that the articles
from my Feedbin account were not loading. When you said “if you ad those
same sources manually,” what did you mean? Because I thought you mean add
RSS feeds not to the sync service, but directly to Fluent Reader.
--

Alex
